datafusion/physical-plan/src/lib.rs

```diff
@@ -455,6 +443,22 @@ impl ExecutionPlanProperties for Arc<dyn ExecutionPlan> {
     ///
     /// See also [`ExecutionPlan::maintains_input_order`] and [`Self::output_ordering`]
     /// for related concepts.
+    fn equivalence_properties(&self) -> &EquivalenceProperties;
```

Comment:
github rendered the diff strangely -- what I did was move the comments into the trait